schemas/context/segmentmembership.schema.json

```diff
@@ -29,6 +29,42 @@
           "format": "date-time",
           "description": "The timestamp when the assertion of segment membership was made."
         },
+        "xdm:previousQualificationTimes": {
+          "title": "Previous Qualification Times",
+          "description": "The timestamp when the assertion of segment membership was made. DO WE WANT TO TRACK PREVIOUS PAYLOADS HERE",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "format": "date-time"
+          }
+        },
+        "xdm:payload" : {
+          "title" : "Payload",
+          "description": "Values that are direclty related with the segment realization. This payload exists with the same validUntil as the segment realization.",
+          "anyOf": [
+            {
+              "type": "string"
+            },
+            {
+              "type" : "number"
+            },
+            {
+              "type" : "object"
+            },
+            {
+              "type" : "array"
+            },
+            {
+              "type" : "boolean"
+            },
+            {
```

Comment:
Do we want to force any structure on the payload? It will be difficult to validate correctly even if we force strict structure because the payload type is ultimately controlled by the schema definition and not really the schema membership. 

I put both ways here for the sake of discussion.